tsconfig.json

```diff
@@ -1,5 +1,6 @@
 {
   "compilerOptions": {
+    "strictNullChecks": true,
```

Comment:
@bzums @Retro64 Should I just create a new issue for the `strictNullCheck` stuff?